buildlib/dockers/ubuntu-release.Dockerfile

```diff
@@ -14,13 +16,15 @@ RUN apt-get update && \
         libcap2 \
         libnuma-dev \
         libtool \
+        libnvidia-compute-${DRIVER_VERSION} \
```

Comment:
Please add a comment why we need libnvidia-compute-${DRIVER_VERSION}

Reply:
libnvidia-compute-<driver_ver> replaces the cuda-compat-<cuda_ver> that comes with nvidia/cuda image by default.
It is a better way to provide dependencies required for building ucx-cuda and allowed us to drop the previous linking hack.

Reply:
I see - please put this comment into the code

Reply:
Added a comment at the Dockerfile.